include/Plugin.h

```diff
@@ -40,8 +42,10 @@ class PixmapLoader;
 class PluginView;
 class AutomatableModel;
 
+class LmmsCore;
```

Comment:
Is this necessary? `LmmsCore` is declared in `Engine.h`, which is being included through `EngineClient`, so I think you can omit the extra forward-declaration.